### PR TITLE
Limit MDX image height

### PR DESCRIPTION
**Pull request by tabathadelane**
_a [theme update](https://github.com/newrelic/gatsby-theme-newrelic/pull/938) will add these features:_
- _Add max height for MDX images at 550px_
- _Unsets this height in lightbox view_
- _An 'unbound' class for `` in MDX files to unset this height restriction where authors deem it is appropriate._

### ✨ Summary

- This PR allow us to pass this className

- It also auto sets the `unbound` className if the `width` is set. this prevents percent values from stretching images. These percent values seem to be used mostly within `collapsers` to keep images small, but in the case that a vertical image hits that max-height, it would be stretched. 

- An extra section was added to the style-guide for tech writers.
 
## 🧪 Steps to reproduce
There are a couple images on this `http://localhost:8000/docs/mlops/integrations/aporia-mlops-integration/` that are larger than 550px. Adding the unbound class will make them full size.

```
<img
  class="unbound"
  title=""
  alt=""
  src={}
/>
```

adding a set width will automatically apply `unbound`
```
<img
  width="80%;"
  title=""
  alt=""
  src={}
/>
```

We should check the site for edge cases. 

### 🖼️ Screenshots

  Screenshot 2023-09-14 at 11 52 44 AM

### 🔮 Engineering checklist

- [x] Relevant documentation including README's or Confluence are updated to reflect these changes
- [ ] Changes that effect tech writers have been communicated to tech docs team
- All downstream dependencies for DaaS have been considered for negative impacts
  - [ ] What's new JSON endpoints
  - [ ] Release notes JSON & EOL pages
- [ ] The build preview has been checked for expected functionality on desktop browsers
- [ ] The build preview has been checked for expected functionality on mobile browsers

### CODE REVIEW

File: src/components/MDXContainer.js
Comment (sunnyzanchi) [inline src/components/MDXContainer.js:40]: hmm would it be better have `unbound` in addition to any className passed in?

```suggestion
        className={cx(props.className, props.width && 'unbound')}
```

**Review by sunnyzanchi — CHANGES_REQUESTED** [review]

**Review by sunnyzanchi — DISMISSED** [review]

**Comment by austin-schaefer** [comment]
@tabathadelane : Is this still a PR we're actively working? If not can we close and reopen the related Jira for the backlog?

**Comment by tabathadelane** [comment]
> @tabathadelane : Is this still a PR we're actively working? If not can we close and reopen the related Jira for the backlog?

Yes, this ticket, NR-118948, is still open (blocked) in our On-Deck list. This works needs to merge in with a theme update, but the theme has been blocked by Gastby 5 upgrades. We are doing work this sprint to to unblock it for UX work. I will get visibility on it with the team.

**Comment by tabathadelane** [comment]
reopening to trigger new checks that we have added since this was created.

**Review by clarkmcadoo — APPROVED** [review]

**Review by sunnyzanchi — DISMISSED** [review]
tested again and looks good to me!

**Comment by tabathadelane** [comment]
[successfully built after cache clear](https://www.gatsbyjs.com/dashboard/34e708d2-68a1-48ca-b66f-d2604890b061/sites/a0b77314-cdea-4790-8fde-7d3457d590ce/pull-requests/6297689c-c59a-48ea-b717-bf2dae18c8f1/builds?loggedin=1&token=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI2M2QxNzU3Ny0zZWVmLTQzMjMtODBlNC1kZmRiYjk0NDNkZmQiLCJlbWFpbCI6InRhYmF0aGFkZWxhbmVAZ21haWwuY29tIiwiaWF0IjoxNzAyMzk5MDc2LCJleHAiOjE3MDI0NDIyNzYsImlzcyI6IkdhdHNieSBJbmMiLCJzdWIiOiJ0YWJhdGhhZGVsYW5lQGdtYWlsLmNvbSJ9.Q1QKwOBGgJU3z4xcK4-HaDjYE6Cs8-JULfEy-9589j_pgh_34WlLIJSyInNbFW-vObCTNgVQYn8EQLc9wOlVQ8i6kMoQEJKb6jnSl3V9BuBf5O0VzWHHQ74PPytLqAxRVNfndrVtnVmhNJqZb3a2MPda5ysIMYz2e52RnyhXhmIGobq3Xvus4utier4M4tZZVOW5SGLCaYIWwV4ex45ckKgfMsiG3E6CX1okGW2n1mre74-q6CHZVpIwbNet0RC-zxxplJUHkwY-Tzwg9MsteyyYA2UjVEELhxK5bIhFBndywVVBk2gOSZEbwJ4zVLcOhRuOYLAYBiTE773hx37v82lGk56yHlv12L7zuEnXzSEVHYD4raskuplODhC6tDlHzcSMQUQOb00qQaZQclOPKXv8fyrSKPI3er3eoQ7PtZtdXQsbTkZZYNvDTrHOMGcLjqsRo684Bp6ozSWXagFSqtu9E4xR25rjNyZdOmOLoAcodSMDq4QFWOsl_gIRrIEYN5vX979BpyTix6EhopT_mshuiXPXeuynZyJ0M1ZSDWvYizwhY832_ofJr-HH57arFssMi3xREMwN8L-XVsjFDopGNHvJ3WLgbFvfRicSTwYcaZNUsog4w9Lhwhfbq0__Ze-EfTKcaqTDkeOHg2lIcVzAPyyfuo4YYvsZ7HfYWx0&expiration=2023-12-26T16%3A37%3A56.117Z&ssoToken=&ssoExpiration=&id=63d17577-3eef-4323-80e4-dfdbb9443dfd&loginRedirectUrl=%2Fdashboard%2F34e708d2-68a1-48ca-b66f-d2604890b061%2Fsites%2Fa0b77314-cdea-4790-8fde-7d3457d590ce%2Fpull-requests%2F6297689c-c59a-48ea-b717-bf2dae18c8f1%2Fbuilds&selectedPlan=&redirectToLeadCapture=false)

**Review by clarkmcadoo — APPROVED** [review]

**Review by sunnyzanchi — APPROVED** [review]

**Pull request merged**
